### PR TITLE
ES localization of ´Loosely coupled architecture´

### PR DESCRIPTION
**Pull request by valeriahhdez**
Add Spanish localization of ´loosely coupled architecture

### Describe your changes
This is a Spanish localization of the CNCF concept of ´loosely coupled architecture´. 

### Related issue number or link (ex: `resolves #issue-number`)
Resolves Issue #610 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/es/loosely-coupled-architecture.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: Arquitectura débilmente acoplada
```
Comment (electrocucaracha) [inline content/es/loosely-coupled-architecture.md:2]: Por mencionar algunas opciones más:
* Arquitectura ligeramente acoplada
* Arquitectura [poco acoplada](https://docs.aws.amazon.com/es_es/prescriptive-guidance/latest/patterns/build-a-loosely-coupled-architecture-with-microservices-using-devops-practices-and-aws-cloud9.html)
* Arquitectura [débilmente acoplada](https://learn.microsoft.com/es-es/azure/architecture/framework/devops/principles#use-loosely-coupled-architecture).

File: content/es/loosely-coupled-architecture.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+status: Completed
+category: Propiedad
+tags: ["fundamental", "architecture", "property"]
+---
+
+
+La arquitectura débilmente acoplada es un tipo de arquitectura en donde los componentes individuales de una aplicación se construyen de manera independiente unos de los otros 
+(es el paradigma opuesto a la [arquitectura fuertemente acoplada](/tightly-coupled-architectures/)). 
+Cada componente, a veces denominado [microservicio](/microservices/),
```
Comment (electrocucaracha) [inline content/es/loosely-coupled-architecture.md:11]: Este término ha sido traducido, pero es necesario actualizarlo para que sea visible y pueda ser referenciado.
```suggestion
Cada componente, a veces denominado [microservicio](/es/microservices/), 
```

File: content/es/loosely-coupled-architecture.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+status: Completed
+category: Propiedad
+tags: ["fundamental", "architecture", "property"]
+---
+
+
+La arquitectura débilmente acoplada es un tipo de arquitectura en donde los componentes individuales de una aplicación se construyen de manera independiente unos de los otros 
+(es el paradigma opuesto a la [arquitectura fuertemente acoplada](/tightly-coupled-architectures/)). 
+Cada componente, a veces denominado [microservicio](/microservices/), 
+está diseñado para realizar una función específica de manera que pueda ser utilizado por cualquier número de otros servicios.
```
Comment (electrocucaracha) [inline content/es/loosely-coupled-architecture.md:12]: Aunque la traducción es válida, me parece que sobran palabras.
```suggestion
está diseñado para realizar una función específica de manera que pueda ser utilizado por cualquier número de servicios. 
```

**Review by electrocucaracha — APPROVED** [review]
Muchas gracias por la traducción de este término. Las recomendaciones que realizo son sugerencias que sería bueno comentarlas durante la reunión semanal.

File: content/es/loosely-coupled-architecture.md (hunk 2 above)
Comment (electrocucaracha) [reply content/es/loosely-coupled-architecture.md:11]: He creado el [cambio](https://github.com/cncf/glossary/pull/1742) necesario para que este término sea visible, espero los cambios sean agregados antes de que este cambio sea aprobado

File: content/es/loosely-coupled-architecture.md (hunk 1 above)
Comment (ramrodo) [reply content/es/loosely-coupled-architecture.md:2]: Se decidió utilizar **débilmente** acoplada

**Comment by ramrodo** [comment]
/lgtm
@92nqb @valeriahhdez could you review/approve this PR, please?

**Review by krol3 — APPROVED** [review]
LGTM, @valeriahhdez great job!!

**Pull request merged**